ddtrace/_logger.py

```diff
@@ -37,7 +37,9 @@ def configure_ddtrace_logger():
 
     """
     ddtrace_logger = logging.getLogger("ddtrace")
-    ddtrace_logger.addHandler(logging.StreamHandler())
+    # don't add a handler if already configured by user
+    if not ddtrace_logger.handlers:
+        ddtrace_logger.addHandler(logging.StreamHandler())
     _configure_ddtrace_debug_logger(ddtrace_logger)
     _configure_ddtrace_file_logger(ddtrace_logger)
```

Comment:
will we have the same issue with the file logger?

Reply:
Yeah but if a customer doesn't explicitly set `DD_TRACE_LOG_FILE` ddtrace won't add the handler. 

So it's like: 
"I configured ddtrace logging, but then also wanted it to send to this file, so I added this envar. " - That makes sense to me. If you add an envar for logging, you expect it do something. It'd be confusing if configuring logging kept the envar from having an effect.

The other scenario I view as: "I configured ddtrace logging, and now it's also just adding its own StreamHandler() by default" - which is probably not what users would want if they already configured it.

Reply:
Added an envar to toggle it